docker-compose.yml

```diff
@@ -11,6 +11,7 @@ volumes:
 services:
   volume:
     image: ${volume}
+    container_name: edgex-volume
```

Comment:
Recommend this be named "edgex-files" for consistency with release docker-compose files
https://github.com/edgexfoundry/developer-scripts/blob/master/compose-files/docker-compose-delhi-0.7.1.yml

Reply:
@tsconn23 I had updated the container_name of volume to "edgex-files". Thanks for your advice.